utils/appliance/endpoints/ui.py

```diff
@@ -23,6 +68,52 @@ def __init__(self, owner):
         self.owner = owner
         self.menu = Menu()
 
+    @cached_property
+    def widgetastic(self):
+        """This gives us a widgetastic browser."""
+        return MiqBrowser(manager.ensure_open(), self)
+
+    def create_view(
+            self, view_class, navigate=False, require_navigate=False, additional_context=None):
```

Comment:
though you still have global things like User or so. The Views take dictionaries anyway